src/VirtualClient/VirtualClient.Common/Telemetry/Serilog/SerilogCsvFileLoggerProvider.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace VirtualClient.Common.Telemetry
+{
+    using System;
+    using System.Collections.Generic;
+    using global::Serilog;
+    using Microsoft.Extensions.Logging;
+    using VirtualClient.Common.Extensions;
+    using ILogger = Microsoft.Extensions.Logging.ILogger;
+
+    /// <summary>
+    /// Provides methods for creating <see cref="ILogger"/> instances that can
+    /// be used to log events/messages to a local csv file.
+    /// </summary>
+    [LoggerSpecialization(Name = SpecializationConstant.StructuredLogging)]
+    public sealed class SerilogCsvFileLoggerProvider : ILoggerProvider
+    {
+        private LoggerConfiguration configuration;
```

Comment:
Not a blocker, but you do not need to have extra line breaks after local member variables. It does not improve readability for the most part.